src/shell-app-system.c

```diff
@@ -87,6 +94,31 @@ static void shell_app_system_class_init(ShellAppSystemClass *klass)
 }
 
 static void
+scan_alias_to_id (ShellAppSystem *self)
```

Comment:
Yes, I'm an idiot and didn't push the fixed up commit that called this.

Fixed.